src/org/loklak/objects/MessageEntry.java

```diff
@@ -564,8 +564,8 @@ public JSONObject toJSON() {
     public Post toJSON(final UserEntry user, final boolean calculatedData, final int iflinkexceedslength, final String urlstub) {
 
         // tweet data
-        this.put(AbstractObjectEntry.TIMESTAMP_FIELDNAME, utcFormatter.print(getTimestampDate().getTime()));
-        this.put(AbstractObjectEntry.CREATED_AT_FIELDNAME, utcFormatter.print(getCreatedAt().getTime()));
+        this.put(AbstractObjectEntry.TIMESTAMP_FIELDNAME, AbstractObjectEntry.utcFormatter.print(getTimestampDate().getTime()));
```

Comment:
yep, I have tried checking the logs again, this error isn't appearing.

I don't understand
> Why wasn't it there earlier?